common/test/acceptance/tests/lms/test_bookmarks.py

```diff
@@ -265,35 +267,40 @@ def test_bookmark_button(self):
             Then I click again on the bookmark button
             And I should see a unit un-bookmarked
         """
-        self._test_setup()
+        self.setup_test()
         for index in range(2):
             self.course_home_page.visit()
             self.course_home_page.outline.go_to_section('TestSection{}'.format(index), 'TestSubsection{}'.format(index))
 
             self._toggle_bookmark_and_verify(True, 'bookmarked', 1)
-            self.bookmarks_page.click_bookmarks_button(False)
+            self.course_home_page.visit()
+            self.course_home_page.outline.go_to_section('TestSection{}'.format(index), 'TestSubsection{}'.format(index))
             self._toggle_bookmark_and_verify(False, '', 0)
 
+    # TODO: TNL-6546: Remove this test
+    def test_courseware_bookmarks_button(self):
+        """
+        Scenario: (Temporarily) test that the courseware's "Bookmarks" button works.
+        """
+        self.setup_test()
+        self.courseware_page.click_bookmarks_button()
+        empty_list_text = ("Use bookmarks to help you easily return to courseware pages. "
+                           "To bookmark a page, click on \"Bookmark this page\" beneath the unit title.")
```

Comment:
@andy-armstrong My suggestions: strike "on" and use "under" instead of "beneath".
"To bookmark a page, click "Bookmark this page" under the unit title."